2022-06-08 20:31:52,521:ERROR:__main__:Dalyba is nulio
Traceback (most recent call last):
  File "c:\Users\Lolita\Desktop\pythonkursas\logging_course\main.py", line 194, in dalyba
    rezultatas = a / b
ZeroDivisionError: division by zero
2022-06-08 20:31:52,526:INFO:__main__:Dalyba: 20 / 0 = None
